starter_code/bin/www

```diff
@@ -41,6 +41,7 @@ server.on('error', error => {
   }
 });
 
-server.listen(process.env.PORT, () => {
+server.listen(3000, () => {
```

Comment:
keep this as `process.env.PORT` and create a `.env` file where you define the `PORT=3000` variable without any spaces between them